webapp/svelte.config.js

```diff
@@ -8,10 +8,9 @@ const config = {
     preprocess: vitePreprocess(),
 
     kit: {
-        // adapter-auto only supports some environments, see https://kit.svelte.dev/docs/adapter-auto for a list.
-        // If your environment is not supported, or you settled on a specific environment, switch out the adapter.
-        // See https://kit.svelte.dev/docs/adapters for more information about adapters.
-        adapter: adapter()
+        adapter: adapter({
+            fallback: '/index.html'
```

Comment:
Does this still works if we have a +page.server.ts somewhere?

Reply:
So in that case, if we happen to need to call an external API, does the API key have to be exposed to client side? I am looking at `$env` and it seems that `$env` only load on *server.ts

Reply:
I think I can accept exposing it for this sprint. We can think about proxy later